### ISSUE TITLE
Allow multiple queries at once

### ISSUE THREAD

**Issue by klmr** [EBISPOT/OLS#100]
I may be overlooking something but there doesn’t seem to be an easy way of querying multiple terms at once. My use-case is that I need to retrieve the labels for a list of GO terms. My current code for this looks as follows:

```bash
for term in $terms; do
    echo -n "$term,"
    curl "http://www.ebi.ac.uk/ols/api/ontologies/go/terms?short_form=$term" \
        -H 'Accept: application/json' \
    | jq  ._embedded.terms[0].label
done > labels.csv
```

However, this results in a lot of requests. It would be faster (and, I’d imagine, more efficient for the server) if I could query multiple terms in one request, and get an array of terms in the output (which already happens, the current result simply always has a single entry):

```bash
curl "http://www.ebi.ac.uk/ols/api/ontologies/go/terms?short_form=$terms" \
    -H 'Accept: application/json' \
| jq  '._embedded.terms[] | [.short_form, .label] | @csv' > labels.csv
```

**Comment by simonjupp** [comment]
Yes, this isn't currently possible but I already have a ticket to support this use case and have started work on it. There will be an OLS release in August that will include this feature.

**Comment by atalagk** [comment]
Hi, can I also stress the importance of this feature to the maximum level possible!
One by one lookups for a long list of terms is just not fast enough, even with our own local instance on a pretty good machine.
